### ISSUE TITLE
Superscripts too low

### ISSUE THREAD

**Issue by Doc73** [psb1558/Junicode-font#185]
In my opinion the superscripts are too low (subscripts apparently are missing).

Here an example with the comparison with another font (Libertinus Serif):
```tex
% !TeX TS-program = xelatex
% !TeX encoding = UTF-8
% !TeX spellcheck = it_IT
% !TeX root = Root.tex

\documentclass[11pt,b5paper,article]{memoir}

\usepackage{fontspec,unicode-math}
\setmainfont[]{Junicode Two Beta}
\usepackage{realscripts}

\begin{document}
Realscripts Junicode 
	T\textsuperscript{exc}
	\textit{T\textsuperscript{exc}
	T\textsubscript{exc}
	\textit{T\textsubscript{exc}}}

{\fontspec{Libertinus Serif}Realscripts Libertinus 
	T\textsuperscript{exc}
	\textit{T\textsuperscript{exc}
	T\textsubscript{exc}
	\textit{T\textsubscript{exc}}}
\end{document}
```

Wich produces
![Schermata del 2023-06-29 16-45-21](https://github.com/psb1558/Junicode-font/assets/1306260/9dd329c4-f538-4d01-92b7-47b6f04df2d3)

By the way, many many thanks for all your effort in developing this excellent, very excellent font!

**Comment by psb1558** [comment]
But `\textsubscript` probably just scales and shifts the letters, which won't look as good as a proper `subs` feature. Can you show (or link to) a sample or two of a linguistic work employing subscript letters?

**Comment by Doc73** [comment]
Not exactly, because `realscripts` package uses OpenType features for both super- and subscripts. In case a font lacks these feature, we can use `\textsuperscript*` or `\textsubscript*` commands.

For example:
```tex
% !TeX TS-program = xelatex
% !TeX encoding = UTF-8
% !TeX spellcheck = it_IT

\documentclass[11pt,b5paper,article]{memoir}

\usepackage{fontspec,unicode-math}
\setmainfont[]{Junicode Two Beta}
\usepackage{realscripts}

\begin{document}
Realscripts Junicode

Super: T\textsuperscript{exc} 

Super Italics: \textit{T\textsuperscript{exc}}

Sub: T\textsubscript{exc}

Sub Starred: T\textsubscript*{exc}

Sub Italics: \textit{T\textsubscript{exc}}
\end{document}
```

**Comment by psb1558** [comment]
This is going to take a while, because there are *lots* of superscripts, and then there are italics to take care of, but the project of regularizing them all is going well:
<img width="229" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/00a3806f-6614-458a-a78c-31db5c164448">

**Comment by marrus-sh** [comment]
I did manage to find a use of subscripts in linguistics, in a morphology description:

![image](https://github.com/psb1558/Junicode-font/assets/3212430/6c1ef83d-eb20-46dd-aa2e-902c92799668)

(from “A Grammatical Sketch of Nxa'amxcin (Moses-Columbia Salish)”, doctoral thesis by Marie Louise Willett, 2003, p. 27)

Here you have uppercase subscripts (placed after the brackets) which are (poorly) distinguished from smallcaps (the abbreviated terms inside the brackets). At least, I’m pretty sure that is what is happening.

Subscripted numbers are common (e.g. C₁ to mean “the first consonant”). I think I’ve seen subscript lowercase i, m, and f similarly used to mean “inital”, “medial”, and “final”. Perhaps also subscripted lowercase greek in a similar fashion (e.g. κ for “coda”) but I don’t have any examples of this off the top of my head. I can't recall ever seeing a subscript in linguistics outside of basic Latin or Greek.

**Comment by Doc73** [comment]
Yes, I'm also looking for examples. In classical philology are often used for manuscripts sigla in critical editions,

**Comment by Doc73** [comment]
I'm sorry. some clicks escaped me

**Comment by psb1558** [comment]
Thanks for the images. I get the kind of context, and I don't think I need to see more. But I would like to know what kind of coverage I need in `subs` ‘Subscripts’. I've already got at least a few Unicode-encoded subscript modifier letters. I'll take a survey and make sure the collection is complete, and include them in the `subs` feature the way I've included superscript modifiers in `sups`.

Then we can see what else might be needed in `subs`.
